### ISSUE TITLE
I want to import shapefiles

### ISSUE THREAD

**Issue by IanHopkinson** [scraperwiki/geojson-tool#17]
There is a pip installable Python library for importing shapefiles:

https://pypi.python.org/pypi/pyshp/1.1.4

**Comment by drj11** [comment]
Yeah, and I've used it for private study. It seems to be pretty reasonable.

Does anybody else want to import shapefiles? Have you even got one interesting example?

**Comment by IanHopkinson** [comment]
I did this:
http://www.ianhopkinson.org.uk/2011/05/more-news-from-the-shed%E2%80%A6/

English Heritage has a bunch of data:
http://www.english-heritage.org.uk/professional/protection/process/spatial-data/

I think this is lower priority than making a autorefreshing OData connector https://github.com/scraperwiki/odata-tool/issues/20

**Comment by drj11** [comment]
What will we call the tool once it does GeoJSON, Shapefile, KML, GeoTIFF, etc, etc?

**Comment by IanHopkinson** [comment]
We should call it... "Geo-Tool"!

**Comment by frabcus** [comment]
"Get geographical data"

**Comment by drj11** [comment]
Example shapefile that can actually be downloaded: http://efele.net/maps/tz/canada/tz_canada.zip

Of note:
- Much of the UK data appears to be behind a registration form.
- shapefiles can be quite big
- they are often distributed as zips (there are 4 or 5 related files), so magic unzipping is required.

**Comment by drj11** [comment]
I found this dataset of 80 catchment area for Nottingham primary schools: http://www.opendatanottingham.org.uk/dataset.aspx?id=76

But the coordinates are in OSGB Easting and Northing. It's needs to be reprojected to some vaguely global grid before it can be displayed. Some hints as to how to do that here:

http://geospatialpython.com/2011/09/map-projections.html
